2024-02-12 11:20:04,046 ERROR: Exception on /users [GET] [in C:\Users\User\Desktop\flask2.3.3-scaffold\venv\lib\site-packages\flask\app.py:1414]
Traceback (most recent call last):
  File "C:\Users\User\Desktop\flask2.3.3-scaffold\venv\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\User\Desktop\flask2.3.3-scaffold\venv\lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\User\Desktop\flask2.3.3-scaffold\venv\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\User\Desktop\flask2.3.3-scaffold\venv\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "C:\Users\User\Desktop\flask2.3.3-scaffold\app\common.py", line 30, in decorated_view
    return fn(*args, **kwargs)
  File "C:\Users\User\Desktop\flask2.3.3-scaffold\app\main\routes.py", line 43, in users
    return render_template('users_vue.html')
  File "C:\Users\User\Desktop\flask2.3.3-scaffold\venv\lib\site-packages\flask\templating.py", line 150, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\Users\User\Desktop\flask2.3.3-scaffold\venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\Users\User\Desktop\flask2.3.3-scaffold\venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\Users\User\Desktop\flask2.3.3-scaffold\venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\Users\User\Desktop\flask2.3.3-scaffold\venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\Users\User\Desktop\flask2.3.3-scaffold\venv\lib\site-packages\flask\templating.py", line 64, in get_source
    return self._get_source_fast(environment, template)
  File "C:\Users\User\Desktop\flask2.3.3-scaffold\venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: users_vue.html
2024-02-12 11:23:22,736 INFO: Scaffolding startup [in C:\Users\User\Desktop\flask2.3.3-scaffold\app\__init__.py:81]
2024-02-12 11:24:28,394 INFO: Scaffolding startup [in C:\Users\User\Desktop\flask2.3.3-scaffold\app\__init__.py:81]
2024-02-12 11:24:48,500 INFO: Scaffolding startup [in C:\Users\User\Desktop\flask2.3.3-scaffold\app\__init__.py:81]
2024-02-12 11:34:04,768 INFO: Scaffolding startup [in C:\Users\User\Desktop\flask2.3.3-scaffold\app\__init__.py:81]
2024-02-12 11:35:05,720 INFO: Scaffolding startup [in C:\Users\User\Desktop\flask2.3.3-scaffold\app\__init__.py:81]
